builtin/bisect--helper.c

```diff
@@ -339,6 +340,55 @@ static int bisect_next_check(const struct bisect_terms *terms,
 	return res;
 }
 
+static int get_terms(struct bisect_terms *terms)
+{
+	struct strbuf str = STRBUF_INIT;
+	FILE *fp = NULL;
+	int res = 0;
+
+	fp = fopen(git_path_bisect_terms(), "r");
+	if (!fp) {
+		res = -1;
+		goto finish;
+	}
+
+	free_terms(terms);
+	strbuf_getline_lf(&str, fp);
+	terms->term_bad = strbuf_detach(&str, NULL);
+	strbuf_getline_lf(&str, fp);
+	terms->term_good = strbuf_detach(&str, NULL);
+
+finish:
+	if (fp)
+		fclose(fp);
+	strbuf_release(&str);
+	return res;
+}
+
+static int bisect_terms(struct bisect_terms *terms, const char *option)
+{
+	if (get_terms(terms))
+		return error(_("no terms defined"));
+
+	if (option == NULL) {
```

Comment:
I think we more often use `!var` than `var == NULL`.

Reply:
If we use `!option` it throws an error.

Reply:
then again this is https://github.com/gitgitgadget/git/commit/450ebb7359ec, which is already in `tt/bisect-in-c`.

Reply:
Feel free to skip this change too.